docs/stacks-guide.md

```diff
@@ -0,0 +1,301 @@
+
+## Install and configure Crossplane
+
+To use Crossplane, we'll need to install and configure it. In this case,
+we want to use Crossplane with a cloud provider, so we'll need to
+configure the provider.
+
+### Install Crossplane
+
+The recommended way of installing Crossplane is by using
+[helm][helm-install]. We can grab the most stable version currently
+available by using:
+
+```
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+For more options for installing, including how to install a more
+bleeding-edge version, or how to uninstall, see the [full install
+documentation][crossplane-install-docs].
+
+### Configure support for your cloud provider
+
+Next we'll set up support for our cloud provider of choice! See the
+provider-specific guides:
+
+* [AWS][aws-setup]
+* [GCP][gcp-setup]
+* [Azure][azure-setup]
+
+Then come back here! Don't worry; we'll still be here when you're ready.
+
+Don't see your favorite cloud provider? [Help us add
+support][provider-stack-developer-guide] for it!
+
+## Install support for our application into Crossplane
+
+Now that we've got Crossplane set up and configured to use a cloud
+provider, we're ready to add support for creating WordPresses! We'll do
+this using a Crossplane Stack. For more information about stacks, see
+the [full Stack documentation][stack-docs].
+
+We can use the [Crossplane CLI][crossplane-cli] to install our stack which adds support for
+Wordpress:
+
+```
+kubectl crossplane stack install 'crossplane/sample-stack-wordpress:latest' 'sample-stack-wordpress'
+```
+
+This pulls the stack package from a registry to install it into
+Crossplane. For more details about how to use the CLI, see the
+[documentation for the CLI][crossplane-cli-docs]. For more details about how stacks work behind
+the scenes, see the documentation about the [stack
+manager][stack-manager-docs] and the [stack
+format][stack-format-docs].
+
+## Create a Wordpress
+
+Now that Crossplane supports Wordpress creation, we can ask Crossplane
+to spin up a Wordpress for us. We can do this by creating a Kubernetes
+resource that our Wordpress stack will recognize:
+
+```
+cat > my-wordpress.yaml <<EOF
+apiVersion: wordpress.samples.stacks.crossplane.io/v1alpha1
+kind: WordpressInstance
+metadata:
+  name: my-wordpressinstance
+EOF
+
+kubectl apply -f my-wordpress.yaml
+```
+
+If the control cluster doesn't recognize the Wordpress instance type, it
+could be because the stack is still being installed. Wait a few seconds,
+and try creating the Wordpress instance again.
+
+### Wait
+
+The Wordpress can take a while to spin up, because behind the scenes
+Crossplane is creating all of its dependendencies, which is a database
+and Kubernetes cluster. To check the status, we can look at the
+resources that Crossplane is creating for us:
+
+```
+# The claim for the database
+kubectl get mysqlinstance
```

Comment:
Might be useful to include namespaces on these as well if we standardize.

Reply:
@suskin we need to change the namespace `mynamespace` to `app-project1-dev` in the gcp guide as well if we want to standardize on that.

Reply:
@hasheddan @suskin we can just omit the namespace for portable classes as well actually. Use whatever is set on kubectl for wordpress stack, wordpress instance and portable classes

Reply:
@muvaf definitely true. However, I think we are trying to standardize on these namespace patterns, and it is more explicit IMO. I could be persuaded though... cc @prasek

Reply:
This should be fixed. I went with the explicit approach so readers can see an example of how to specify a namespace they want.